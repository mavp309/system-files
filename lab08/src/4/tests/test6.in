-6.78 -2.22 0.8 5.5
