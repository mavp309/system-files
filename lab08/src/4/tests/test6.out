-4.0300 -1.8200
